### PR TITLE
Commutative w code

### PR DESCRIPTION
**Pull request by retabak**
sorry, forgot to push some things on the last PR. Here's a new one!

### CODE REVIEW

**Review by schanzer — CHANGES_REQUESTED** [review]
Overall looking good. One item to change, which we should discuss in our meeting

**Comment by retabak** [comment]
@schanzer - I believe that I've responded to your feedback. Take a look and let me know if this looks ready to merge!

File: lessons/Early-Math/commutative-property/langs/en-us/index.adoc [hunk 1]
```diff
@@ -193,7 +202,7 @@ When everyone is finished, check in with students. Did everyone discover that _n
 @slidebreak
 
 
-In the last activity, we discovered that _none_ of the functions on the page were commutative! Do you think there are any functions in @proglang that are commutative? Let's answer that question.
+In the last activity, we discovered that _none_ of the image-producing functions on the page were commutative! Do you think there are any functions in @proglang that are commutative? Let's answer that question.
```
Comment (schanzer) [inline lessons/Early-Math/commutative-property/langs/en-us/index.adoc:205]: You're asking them to answer a simple yes-or-no question, when thinking about the actual answer would be nearly impossible without having more examples of image-producing functions. That means students will just flip a coin and pick "yes" or "no", neither of which is helping them think through stuff.

I'm sorry to disappoint you, but now I'm even more convinced that this section is backwards. Here's how I would set it up:

- Give kids a workbook page with `above`, `beside`, and `overlay`. For each one, have them read the contract, convert a given COE to code, then draw the COE with arguments swapped. Students have to make a prediction about whether they will get the same value back by _drawing the image they predict they'll get back_ when they convert the swapped COE to code. Have them try it out, draw the answer, then circle "commutative" or "not commutative".  At the bottom, leave an open response that asks them if they can imagine an image function that IS commutative
- Page 2: Same formula, but with`string-contains`, `min`, `max`, and `blend-images`

File: lessons/Early-Math/commutative-property/langs/en-us/pages/commutativity-and-code2.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ Open the @starter-file{comm-and-assoc}. Complete the exploration to determine if
 
 @fitb{}{}
 
-@n Use @proglang to test your prediction. What did you discover? @fitb{}{}
+@n Use @proglang to test your prediction. What did you discover? @fitb{}{@ifsoln{`max` is commutative.}}
```
Comment (schanzer) [inline lessons/Early-Math/commutative-property/langs/en-us/pages/commutativity-and-code2.adoc:43]: I also think that asking them to do so much writing is overkill, especially given that our audience is younger and this is literally a yes/no answer anyway

**Review by schanzer — CHANGES_REQUESTED** [review]
Don't kill me -- I liked your justification when you explained it, but after reading the code section again I am even more convinced it doesn't hang together. I left you some very concrete feedback, which I hope (a) makes sense and (b) is helpful.

File: lessons/Early-Math/commutative-property/langs/en-us/index.adoc [hunk 3]
```diff
@@ -175,14 +175,23 @@ But how about functions in @proglang?!
 
 @slidebreak
 
-As a programmer, you will definitely want (and need!) to know if you can change around the order of a function's arguments... or if you need to adhere to just one single order. Let's dig in!
+As a programmer, you will definitely want (and need!) to know if you can change around the order of a function's arguments... or if you need to adhere to just one single order.
 
 === Investigate
 
+@QandA{
+Think about the function `beside`, which takes in two images - let's say a green star and a blue square - and places them one next to the other.
```
[the remark this answers is not in the document]
Comment (retabak) [reply lessons/Early-Math/commutative-property/langs/en-us/index.adoc:183]: They do! The Contacts lesson is in early math.

[the remark this answers is not in the document]
Comment (schanzer) [reply lessons/Early-Math/commutative-property/langs/en-us/index.adoc:183]: That lesson does not cover `beside`. In fact, it doesn't cover any of the image functions you use here.

Open the lesson plan and do a search for "beside".

**Pull request closed without merging**